src/lib/comms/sol-http-server-impl-microhttpd.c

```diff
@@ -52,6 +52,21 @@
 
 #define SOL_HTTP_REQUEST_BUFFER_SIZE 4096
 
+#ifndef SOL_NO_API_VERSION
+#define SOL_HTTP_SERVER_CONFIG_CHECK_API_VERSION(config, ...) \
+    if (!(config)->api_version) { \
```

Comment:
Shouldn't this be  `if  ((config)->api_version != SOL_HTTP_SERVER_CONFIG_API_VERSION)`?